### PR TITLE
Update node.go

### PR DESCRIPTION
**Pull request by guoguodan**
it may be null point,  add null point check.

### CODE REVIEW

File: pkg/yurtctl/cmd/convert/node.go [hunk 1]
```diff
@@ -146,18 +146,24 @@ func (c *ConvertNodeOptions) SetupYurthub(workingMode util.WorkingMode) error {
 	if err != nil {
 		return err
 	}
-	kubeletConfPath = strings.Split(kubeletConfPath, "=")[1]
-	apiserverAddr, err := enutil.GetSingleContentFromFile(kubeletConfPath, apiserverAddrRegularExpression)
+	kubeletConfPathArr := strings.Split(kubeletConfPath, "=")
+	if len(kubeletConfPathArr) < 2 {
```
Comment (Peeknut) [inline pkg/yurtctl/cmd/convert/node.go:150]: I think  const var `kubeletConfigRegularExpression` used in `kubeletConfPath, err := enutil.GetSingleContentFromFile(c.KubeadmConfPath, kubeletConfigRegularExpression)` can guarantee this situation.

File: pkg/yurtctl/cmd/convert/node.go [hunk 2]
```diff
@@ -146,18 +146,24 @@ func (c *ConvertNodeOptions) SetupYurthub(workingMode util.WorkingMode) error {
...
+		return fmt.Errorf("--kubeconfig-path format error")
+	}
+	apiserverAddr, err := enutil.GetSingleContentFromFile(kubeletConfPathArr[1], apiserverAddrRegularExpression)
 	if err != nil {
 		return err
 	}
-	apiserverAddr = strings.Split(apiserverAddr, " ")[1]
+	apiserverAddrArr := strings.Split(apiserverAddr, " ")
+	if len(apiserverAddrArr) < 2 {
```
Comment (Peeknut) [inline pkg/yurtctl/cmd/convert/node.go:158]: same as before.

Comment (guoguodan) [reply pkg/yurtctl/cmd/convert/node.go:158]: ok  thinks

**Pull request closed without merging**